joomla/joomla-cms
# [Isis] Template.less refactoring

Pull Request for Issue # .

### Summary of Changes
This PR starts a process of refactoring the Isis template.less, proposing to split the template.less in separate less files. Currently the template.less looks like a long list of PRs with no real structure (with the exception of its initial creation). Breaking it in to its separate components (blocks) makes for more manageable code.

```
blocks
| _global.less
| _chzn-override.less
| _editors.less
| _forms.less
| _header.less
| _login.less
| _media.less
| _modals.less
| _navbar.less
| _quickicons.less
| _sidebar.less
| _status.less
| _tables.less
| _toolbar.less
| _treeselect.less
| _utility-classes.less
| _custom.less
pages
| _com_cpanel.less
| _com_postinstall.less
| _com_templates.less
template.less
```

### Testing Instructions
Apply patch a check admin for issues. Although I suspect not, it may cause some cascading issues and would require a thorough testing.
